stories/Popover/Popover.stories.tsx

```diff
@@ -27,97 +65,267 @@ const ElementiDisabilitatiComponent = () => {
+          >
+            Popover {'a destra'}
+          </Button>
+          <Popover
+            placement={'right'}
+            isOpen={popoverOpen2}
+            target={targetRef2}
+            toggle={() => {
+              setPopoverOpen2(!popoverOpen2);
+            }}
+          >
+            <PopoverBody>
+              Vivamus sagittis lacus vel augue laoreet rutrum faucibus
+            </PopoverBody>
+          </Popover>
+        </span>
+      </div>
+      <div className='mx-auto align-self-center p-2'>
+        <span>
+          <Button
+            className='mr-1'
+            color='secondary'
+            innerRef={targetRef3}
+            onClick={() => {
+              setPopoverOpen3(!popoverOpen3);
+            }}
+          >
+            Popover {'in alto'}
+          </Button>
+          <Popover
+            placement={'top'}
+            isOpen={popoverOpen3}
+            target={targetRef3}
+            toggle={() => {
+              setPopoverOpen3(!popoverOpen3);
+            }}
+          >
+            <PopoverBody>
+              Vivamus sagittis lacus vel augue laoreet rutrum faucibus
+            </PopoverBody>
+          </Popover>
+        </span>
+      </div>
+      <div className='mx-auto align-self-center p-2'>
+        <span>
+          <Button
+            className='mr-1'
+            color='secondary'
+            innerRef={targetRef4}
+            onClick={() => {
+              setPopoverOpen4(!popoverOpen4);
+            }}
+          >
+            Popover {'in basso'}
+          </Button>
+          <Popover
+            placement={'bottom'}
+            isOpen={popoverOpen4}
+            target={targetRef4}
+            toggle={() => {
+              setPopoverOpen4(!popoverOpen4);
+            }}
+          >
+            <PopoverBody>
+              Vivamus sagittis lacus vel augue laoreet rutrum faucibus
+            </PopoverBody>
+          </Popover>
+        </span>
+      </div>
+    </div>
+  );
+};
+export const _LeQuattroDirezioni = PopoverPositionExample.bind({});
+_LeQuattroDirezioni.story = {
   name: 'Le quattro direzioni'
 };
 
-export const TitoloConIconaELink = PopoverIconLink;
+const PopoverIconLink = () => {
+  const [popoverOpen, setPopoverOpen] = useState(false);
+  const targetRef = useRef(null);
+
+  const togglePopover = () => {
+    setPopoverOpen(!popoverOpen);
+  };
+  return (
+    <div>
+      <Button
+        color='secondary'
+        data-container='body'
+        data-toggle='popover'
+        data-placement='right'
+        data-html='true'
+        innerRef={targetRef}
+        onClick={togglePopover}
+      >
+        Popover con icona e link
+      </Button>
 
-TitoloConIconaELink.story = {
+      <Popover
+        placement='right'
+        target={targetRef}
+        isOpen={popoverOpen}
+        toggle={togglePopover}
+      >
+        <PopoverHeader>
+          <Icon color='white' icon='it-help-circle' aria-hidden />
+          Titolo con icona
```

Comment:
I think the icon color has been set to `white` by mistake. Can you remove that color prop?

Current storybook:
<img width="518" alt="Screenshot 2021-05-25 at 08 22 57" src="https://user-images.githubusercontent.com/924948/119450120-c7eb4b00-bd33-11eb-8633-b1f89dba2a7d.png">

Not a major problem, but it would be nice to get it fixed.

Reply:
Correct, removed :+1: